src/models/base/ScholarshipModel.ts

```diff
@@ -0,0 +1,13 @@
+// import firebase from 'firebase/app';
+import FirestoreModel from './FirestoreModel';
+
+export default class ScholarshipModel<
+  T extends { lastModified?: Date; dateAdded?: Date }
```

Comment:
without this it was complaining that `dateAdded` and `lastModified' did not exist on type T